.github/SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| > 2.5.0 | :white_check_mark: |
+
+## Reporting Vulnerabilities
+
+WSO2 takes security issues very seriously. If you have any concerns regarding our product security or have uncovered a security vulnerability, we strongly encourage you to report that to our private and highly confidential security mailing list: security@wso2.com first, without disclosing them in any forums, sites or other groups - public or private. To protect the end-user security, these issues could be disclosed in other places only after WSO2 completes its Vulnerability Management Process.
```

Comment:
Shall we use HTML <a> Tag for "Vulnerability Management Process" with https://docs.wso2.com/display/Security/WSO2+Security+Vulnerability+Management+Process

Reply:
Fixed